week4/assignment/src/components/Header.tsx

```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from "react-router";
+import { useAuth } from "../contexts/AuthContext";
+import userIcon from "../assets/user-regular.svg";
+
+const Header = () => {
+  const nav = useNavigate();
```

Comment:
```suggestion
  const navigation = useNavigate();
```

nav는 <nav>예약어가 이미 존재하기 때문에 혼동을 야기할 수 있어보여요.
정훈님이 변수를 최대한 간단하게 선언하는걸 선호하는 것 같은데, 저는 길어지더라도 의미를 충분히 담을 수 있는 변수 네이밍이 더 좋다고 생각합니다.